DETAILED_INSTALL.md

```diff
@@ -203,6 +203,17 @@ docker run --gpus all -it --rm  -v $(pwd):/home/app/mava -w /home/app/mava insta
 
         If this fails, follow instructions [here](https://github.com/deepmind/meltingpot#installation).
 
+    * Development
+ 
+        When developing mava locally one can just replace the `id-mava` part of the `pip install` with a `.`.  
+        For example when installing petting zoo dependencies for local development (inside of the mava folder):
```

Comment:
I think we can simplify this part, the "inside of the mava folder" might not be clear for new users. 

Maybe rephrase to something like,
For example, if you would to install a local version of mava, including the petting zoo dependencies...

I also think the content added to the readme, should be placed here. We should keep the readme simple and here we can we can keep all the instructions to development on mava. Maybe we can also link to this page in the Contributing docs.

Reply:
Agreed, I'll remove it from the readme and I'll added a section in the contribution docs with a similar example